### ISSUE TITLE
✅ 🚀 Feature Request: Hide some elements/widgets from Elemenor - [ED-12546]

### ISSUE THREAD

**Issue by wayheming** [elementor/elementor#10809]
Hello, such a question has already been asked in this [issue](https://github.com/elementor/elementor/issues/7670#issuecomment-478331892) and @shilo-ey has already given the answer and solution worked perfectly up to this [commit,](https://github.com/elementor/elementor/commit/779175a646fb8cf945f9dc00dd921d62dbe590f8) namely deleting this [line](https://github.com/elementor/elementor/commit/779175a646fb8cf945f9dc00dd921d62dbe590f8#r37576023). Could you give a new instruction?

**Comment by clemensmol** [comment]
I found a solution, works on the latest version and current master branch. You might want to add some code for backwards compatibility. Not needed in my case.


Hide HTML widget:
```php
add_filter('elementor/document/config', function()
{
  $widgetConfig = [
    'html' => [
      'show_in_panel' => 0
    ]
  ]
  return ['widgets' => $widgetConfig];
});
```


Hide all widgets except whitelisted widgets:
```php
add_filter('elementor/document/config', function()
{
  $widget_whitelist = ['form', 'video'];

  $widget_manager = \Elementor\Plugin::instance()->widgets_manager;
  $widget_types_config = $widget_manager->get_widget_types_config();
  $widgetConfigOverrides = [];
  foreach($widget_types_config as $type=>$widget)
  {
    if(!in_array($type, $widget_whitelist))
    {
      $widgetConfigOverrides[$type] = ['show_in_panel' => 0];
    }
  }

  return ['widgets' => $widgetConfigOverrides];
});
```

**Comment by wayheming** [comment]
@clemensmol Thank you for your answer, but I wonder if the team deleted the old method, then maybe they created a new more convenient.

**Comment by wayheming** [comment]
@KingYes @shilo-ey @lior-sh can someone help me?

**Comment by nicholaszein** [comment]
Hello! 👋 

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.17.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
